static/css/styles.css

```diff
@@ -351,22 +318,137 @@ aside#overlay.hide {
     }
 }
 
-.navbar-toggler {
-    color: white;
+/* Page Settings */

```

Comment:
Please don't override the global styles and bypass all of the Bootstrap content globally. Target individual items where necessary - please remove the global styles below that override everything (and use lowercase element targets, example `p` in lieu of `P`)

Reply:
I'm completely lost on this one :'(